group_vars/mainnet.yml

```diff
@@ -11,3 +11,4 @@ starknet:
     staking_contract_address: "0x00ca1702e64c81d9a07b86bd2c540188d92a2c73cf5cc0e508d949015e7e84a7"
     attestation_contract_address: "0x10398fe631af9ab2311840432d507bf7ef4b959ae967f1507928f5afe888a99"
     staker_operational_address: "0x03f3679c2dd729d4f2e22db68d52b1a9defa2246e9fcc4bc644e47801c845c81"
+  account_address: "0x03f3679c2dd729d4f2e22db68d52b1a9defa2246e9fcc4bc644e47801c845c81"
```

Comment:
Do we need a separate variable for this ? considering staker_operational_address is already there